web/src/scripts/containers/TabPane.jsx

```diff
@@ -105,12 +105,12 @@ class TabPane extends Component {
     const {tabIds = []} = this.props
 
     return tabIds.map((tabId) => {
-      const name = ContentLoader.getResourceName(tabId)
+      const {name, path} = ContentLoader.getResourceNameAndPath(tabId)
 
       return (
         <Tab
           key={tabId}
-          title={name}
+          title={path}
```

Comment:
I thought the same thing at first and changed it, but the way Tab actually uses title is exactly that, as a title. So, it's a title in the sense that it's what gets shown on hover and used for nothing else. That makes me think the name is actually accurate.